src/password-store.sh

```diff
@@ -588,6 +586,8 @@ cmd_git() {
 		git_add_file .gitattributes "Configure git repository for gpg file diff."
 		git config --local diff.gpg.binary true
 		git config --local diff.gpg.textconv "$GPG -d ${GPG_OPTS[*]}"
+	elif [[ $1 == "submodule" ]]; then
+		git -C "$GIT_WORK_TREE" "$@"
```

Comment:
I see.

Is `-C` supported by all versions of Git that pass supports?
If so, why isn't it used instead of `GIT_*` environment variables?